h2o-r/h2o-package/R/explain.R

```diff
@@ -2762,8 +2838,13 @@ h2o.ice_plot <- function(model,
                          show_pdp = TRUE,
                          binary_response_scale = c("response", "logodds"),
                          centered = FALSE,
-                         grouping_column = NULL) {
-  return(pd_ice_common(model, newdata, column, target, NULL, max_levels, show_pdp, binary_response_scale, centered, TRUE, grouping_column))
+                         grouping_column = NULL,
+                         output_graphing_data = FALSE,
+                         nbins = 100,
+                         ...) {
+  kwargs <- list(...)
+  grouping_variable_value = kwargs$grouping_variable_value
```

Comment:
If possible please use `<-` for the assignment.

And be careful about using `$` for accessing elements of a list. This method is mainly for interactive purposes, it can do some unexpected things like prefix matching:

```r
> lst <- list(graphing_variable_value=42, graphing_variable=NULL)
> lst$graphing_variable
NULL
> lst$graphing_variable <- NULL
> lst$graphing_variable
[1] 42
> lst[["graphing_variable"]]
NULL
```

Preferred alternative is to use the `[[str]]`.

Reply:
[a9dfa0d](https://github.com/h2oai/h2o-3/pull/6119/commits/a9dfa0dca78c37d998caa4dbd8d975cb361b9ea8)